roles/notifications/files/notification-startup.service

```diff
@@ -0,0 +1,5 @@
+[Unit]
+Description=Notify Telegram channel, server has started running
+
+[Service]
+ExecStart=/usr/local/bin/notification-startup.sh
```

Comment:
Shouldn't it be better to use an `After=${some_service}` directive instead of using a timer? Also, since this is a service that runs and then dies, I think it should use the `Type=oneshot`

Reply:
It repeats daily and thus should be a timer.